tests/notifications/test_creator_notifications.py

```diff
@@ -87,3 +87,85 @@ def test_notify_creator_on_moderator_feedback(proposal_factory, client):
     assert mail.outbox[1].to[0] == creator.email
     assert mail.outbox[1].subject.startswith(
         'Feedback for your contribution')
+
+
+@pytest.mark.django_db
+def test_notify_comment_creator_on_block(
+        idea, comment_factory, apiclient, admin,
+        ai_classification_factory):
+    """Check if creator gets email on comment blocked."""
+    comment = comment_factory(content_object=idea)
+    ai_classification_factory(comment=comment)
+    creator = comment.creator
+    # FIXME: use moderator instead of admin when rules are there
+    # moderator = comment.project.moderators.first()
```

Comment:
@Rineee If we merge your's first, I can do this. Otherwise, could you add the fix to your PR?

Reply:
I'll do it in mine!